examples/platform/silabs/SiWx917/device/inc/system_si917.h

```diff
@@ -0,0 +1,169 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#ifndef SYSTEM_EFR32_H
+#define SYSTEM_EFR32_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "core_cm4.h" /* Cortex-M4 processor and core peripherals */
+#include <stdint.h>
+//#include "cmsis_gcc.h"
+
+/***************************************************************************/ /**
```

Comment:
the `*/  /*` mess up the styling. please remove

Reply:
done on the latest commit